### PR TITLE
Stores the ICE servers configuration in local storage (see #959).

### PR DESCRIPTION
**Pull request by ErikHellman**
**Description**
Storing the configured ICE servers in localStorage to make testing easier. 
Added a reset button to revert to the original configuration.

See #959

### CODE REVIEW

File: src/content/peerconnection/trickle-ice/js/main.js [hunk 1]
```diff
@@ -32,6 +41,35 @@ let begin;
...
+
+function readServersFromLocalStorage() {
+  document.querySelectorAll('select#servers option').forEach(option => option.remove());
+  const serversSelect = document.querySelector('select#servers');
+  const storedServers = window.localStorage.getItem(allServersKey);
+
+  if (storedServers === null || storedServers === '') {
+    const o = document.createElement('option');
+    o.value = '{"urls":["stun:stun.l.google.com:19302"]}';
+    o.text = 'stun:stun.l.google.com:19302';
```
Comment (alvestrand) [inline src/content/peerconnection/trickle-ice/js/main.js]: You don't want all these specific values in two places. Make a function or a constant for "setDefaultServers" and call it from both places.

File: src/content/peerconnection/trickle-ice/js/main.js [hunk 2]
```diff
@@ -32,6 +41,35 @@ let begin;
...
+    serversSelect.add(o);
+  } else {
+    JSON.parse(storedServers).forEach((server, key) => {
+      const o = document.createElement('option');
+      o.value = JSON.stringify(server);
+      o.text = server.urls[0];
+      o.ondblclick = selectServer;
```
Comment (alvestrand) [inline src/content/peerconnection/trickle-ice/js/main.js:68]: Not in this CL (it was there already), but how discoverable is a dobule-click interface?
What's it supposed to do?

**Review by alvestrand — APPROVED** [review]
Looks good, with one nit.

File: src/content/peerconnection/trickle-ice/js/main.js (hunk 2 above)
Comment (ErikHellman) [reply src/content/peerconnection/trickle-ice/js/main.js:68]: Agree. It seems it was added to quickly add new servers. I'll remove this for now as it is just confusing.

**Pull request merged**